### PR TITLE
0.22 release

### PR DESCRIPTION
**Pull request by adrinjalali**

### CODE REVIEW

**Comment by Kapeli** [comment]
You need to update the actual docset as well (`scikit-learn.tgz`).

**Comment by adrinjalali** [comment]
ok, but why close the PR?

**Comment by Kapeli** [comment]
> ok, but why close the PR?

Why not?

**Comment by adrinjalali** [comment]
cause I can apply your requested changes in this PR, and once it's at a stage it's acceptable, you can merge.

**Comment by Kapeli** [comment]
Please open a new pull request once it's ready. Sorry, I try to keep the number of open pull requests at 0.

**Comment by adrinjalali** [comment]
sorry, busy for now.

**Comment by Kapeli** [comment]
> sorry, busy for now.

That kind of makes my point for not keeping pull requests open 👍

**Comment by adrinjalali** [comment]
What I meant to say was that this just discourages me from doing it, because I don't understand why you would take a dogmatic approach and keep your PR count at 0 at the cost of repelling contributors.

I would be very happy to contribute in a reasonable system, but this is far from that. I was just being polite.

**Comment by jnothman** [comment]
@Kapeli, your readmes should really specify that a contributor is expected to upload a new rendered version of the docset. At the moment the main repo readme doesn't say that at all, and the docset readme certainly doesn't.

Contributors who are otherwise used to deployment tools that involve automation usually expect that updating the version number will result in a rebuild.

**Comment by Kapeli** [comment]
@jnothman As far as I remember this has never been an issue. This might be the first pull request that just changes a version number and nothing else.

I'd like to keep the main README as short and concise as possible, as the instructions are already a bit hard to follow. If you find a way to add this information without making things more confusing, please open a pull request.

The docset READMEs are each contributor's responsibility. I provide a sample README which some contributors follow and others don't, which I agree is not ideal.
